docs/source/_json/workflow_get.json

```diff
@@ -0,0 +1,27 @@
+GET /plone/front-page/workflow
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+{
+  "history": [
+    {
+      "action": null, 
+      "actor": "test_user_1_", 
+      "comments": "", 
+      "review_state": "private", 
```

Comment:
Can the current workflow state only be determined by looking at the most recent history entry? Or am I missing something?